app/src/main/res/layout/activity_partners.xml

```diff
@@ -30,13 +30,15 @@
         android:layout_width="match_parent"
         android:layout_height="wrap_content">
 
-        <ImageButton
-            android:layout_width="50dp"
-            android:layout_height="50dp"
+        <EditText
+            android:id="@+id/searchEditText"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
             android:layout_alignParentTop="true"
             android:layout_alignParentLeft="true"
-            android:onClick="enter"
-            app:srcCompat="@android:drawable/ic_menu_revert" />
+            android:maxLines="1"
+            android:singleLine="true"
+            android:hint="Partner suchen" />
```

Comment:
Even it was already a hard-coded string in the original code, it would be better to replace it with a resource string. Sooner or later we will translate the app and than we have to fix that anyway.